app/models/user.rb

```diff
@@ -12,6 +12,10 @@ class User < ApplicationRecord
   # Assign an API key
   after_initialize :add_key, on: :create
 
+  def self.system_user
+    new(username: Chipmunk.config["admin_username"], email: Chipmunk.config["admin_useremail"])
```

Comment:
I think a literal `(system)` is better for the username here. It's already a bit of a concession that we have a fake user and it must have a username/email, rather than a proper notion of the system taking action. I cannot imagine a scenario where that username should be configured. It is "the system", not an unnamed administrator.

The email key here is mismatched with the config file. I think `admin_email` like it is in the config file is fine as it stands. In the grand scheme, I'd prefer to formalize a notification scheme and recipients, but this has to do for now.